.gitignore

```diff
@@ -1,13 +1,18 @@
 .improbable/
 .idea/
 spatialos.*.build.json
-packages/
 logs/
 
-SpatialOS/schema/bin/
+SpatialOS/schema_out
+SpatialOS/tools/
 OtherWorkers/Interactive/client/bin/
 OtherWorkers/Interactive/client/src/improbable/
```

Comment:
I think you can remove the `OtherWorkers` folder references as it should't exist anymore in this refactor

Reply:
addressed by merging the branch #14